mno_analysis_tools/compute_window_of_downtime.py

```diff
@@ -0,0 +1,89 @@
+import json
+import pytz
+import argparse
+from datetime import datetime
+
+from temba_client.v2 import Message
+from core_data_modules.logging import Logger
+
+
+log = Logger(__name__)
+log.set_project_name("ComputeWindowOfDowntime")
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Compute maximum window of time with 0 messages")
+    parser.add_argument("input_file", metavar="input file", type=argparse.FileType(mode="r"),
+        help="File to read the raw data downloaded as json.",
+    )
+    parser.add_argument("output_file", metavar="output file", type=argparse.FileType(mode="w"),
+        help="File to write the raw data downloaded as json.",
+    )
+    parser.add_argument("communication_medium", metavar="operator",
+        help="Operator that you'll need to analyze",
+    )
+    parser.add_argument("direction", metavar="direction of message",
+        help="Direction of the message either sent or received by the operator",
+    )
+    parser.add_argument("start_date", metavar="start date", 
+        type=lambda s: datetime.strptime(s, '%Y-%m-%d').astimezone(pytz.UTC),
```

Comment:
It's convenient to be able to specify a timezone in the country we're operating in. It's also good to catch people who are attempting to use local time rather than assume UTC. Please use isoparse() here instead.